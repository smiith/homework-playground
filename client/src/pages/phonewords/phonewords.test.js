import {shallow} from 'enzyme'
import React from 'react'

describe('Profile', () => {
	it('render single', () => {
		expect(shallow(<div/>)).toMatchSnapshot()
	})
})
