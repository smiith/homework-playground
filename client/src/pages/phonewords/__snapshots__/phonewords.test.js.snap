// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile render single 1`] = `<div />`;
